libs/ui/src/bmd_paper_ballot.tsx

```diff
@@ -174,7 +174,7 @@ const Ballot = styled.div<BallotProps>`
         /* istanbul ignore next - hardware specs still in flux */
         case 'bmd150':
           /* istanbul ignore next - hardware specs still in flux */
-          return '7.975in 11in';
+          return '7.975in 13.25in';
```

Comment:
Out of curiosity, why do we use 7.975in instead of 8in for the height here and in the printing dimensions?

Reply:
8in results in a width of exactly 1600 dots. The math is correct but for some reason 1600 dots overflows when printing on actual hardware. The extra dots overflow to the next line and cause a mostly blank line to be printed, resulting in stripes in the printed document.

I haven't had a chance to find the root cause yet unfortunately.

Reply:
Weird! Maybe leave a comment explaining this issue?